cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,36 @@
-describe('', () => {
-  before(() => {
+const { generateAticle } = require('../support/genArticle');
 
+describe('test for article flow', () => {
+  
+  beforeEach(() => {
+    cy.register().then(({username,email,password}) => {
+      cy.login({ username, email, password });
+    });
   });
 
-  it('', () => {
+  it.only('should be able to create a new article', () => {
+    const {title, description, body} = generateAticle();
+    cy.visit('editor')
+    cy.get('input[placeholder="Article Title"]')
+      .type(title);
+    cy.get('input[placeholder="What\'s this article about?"]')
+      .type(description);
+    cy.get('textarea.form-control[placeholder="Write your article (in markdown)"]')
+      .type(body);
+    cy.get('.btn').click();
+    cy.get('h1').should('contain.text', title)
 
   });
 
-  it('', () => {
+  it('should be able to delete article', () => {
+    const {title, description, body} = generateAticle();
+    cy.createArticle(title, description, body).then(({slug,username }) => {
+      cy.visit(`/article/${slug}`)
+      cy.contains('button.btn.btn-outline-danger.btn-sm', 'Delete Article').click()
+      cy.visit(`profile/${username}`)
+      cy.get('.article-preview').should('contain.text', 'No articles are here... yet.')
```

Comment:
Imagine that you have more than 2 articles on your account, then this test will fail. Try to delete an article then visit the article page and check that there is an error that the article has not found. Like this 
![image](https://github.com/mate-academy/qa_cypress_e2e_article_flow/assets/85580286/55f09800-3c2a-4577-a0f2-4d7b04a6c3f1)

Reply:
I can't find a trigger to get a 404 window, even with 20 sec waiting after deleting the article, with page refreshing, deleting cookies\local storage. The only way I have found is to check if API sends 204 response code

Reply:
Your option is also good